app/src/main/sentences/sl/navigation.dslf

```diff
@@ -1,3 +1,3 @@
 navigation: high
-(pelji|zapelji me?)|((poišči mi?)? navodila<s?>?) v|čez? .where.;
+pelji|zapelji me |navodila v|čez|za|na? .where.;
```

Comment:
Are you sure this is correct? The `|` is used to define alternatives that (semantically) mean the same thing. But to me it doesn't seem like `me` and `navodila` are alternative, since the first means "to me" an the other means "instructions".

On a side note, when dealing with these sentences, the space character has the only purpose of separating words, but has no meaning per se. A word is a contiguous set of letters. So `a|b c|d`. `a |b c   |d` and `(a | b)(c |d)` are all the same thing and they all match all of "a c", "a d", "b c", "b d". In the third option you can notice how you don't necessarily need a space to separate words, since any non-letter character would do (a `)`, in this case), but while the space does not have any meaning and can be put anywhere, other characters such as `|`, `?`, `)` and `(` have a meaning.
Parentheses can be used to group together part of a sentence that needs to be treated as a single object. So `a|(b c)|d` is different from the strings above, and matches all of "a", "b c" and "d".

The sentence may be a good candidate, but I am not sure, so please confirm. Can the `navodila` be marked as optional or should it always be present?
```suggestion
pelji|zapelji me? navodila v|čez|za|na? .where.;
```

This is an interpretation of the sentence I suggested:
START WITH "pelji" OR "zapelji", THEN EITHER "me" OR NOTHING, THEN "navodila", THEN ONE OF "v", "cez", "za", "na" OR NOTHING, THEN capture everything that follows into the "where" capturing group